### ISSUE TITLE
[Feature] <adding suggestions to search bar>

### ISSUE THREAD

**Issue by tarunprakashporam** [rupali-codes/LinksHub#600]
### Description

I want to add the nearest word suggestions to the search bar , can you please add it to the GSSOC'2023

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am a GSSoC'23 contributor

**Comment by rupali-codes** [comment]
@tarunprakashporam Hey, but we already have search bar?

**Comment by tarunprakashporam** [comment]
while typing the word , we can add suggestions to the search bar

**Comment by rupali-codes** [comment]
> while typing the word , we can add suggestions to the search bar

Sounds cool, assigning to you :)

**Comment by Phyozawhein** [comment]
Hi,
I am interested in this issue. May I get assigned to this so that I can work on it ?

**Comment by CBID2** [comment]
> Hi,
> 
> I am interested in this issue. May I get assigned to this so that I can work on it ?

Done @Phyozawhein!

**Comment by Phyozawhein** [comment]
Hi All, 
I just followed the issues and this is what I have learnt and I want to confirm with you all : The drop down suggestion has been created but the suggestions are NOT clickable yet. So the following feature needs to be added: when a suggestion is clicked, the right section will render the clicked suggestion's page.

**Comment by CBID2** [comment]
> Hi All, I just followed the issues and this is what I have learnt and I want to confirm with you all. The following features need to be added:
> 
> 1. Create a drop down suggestion list
> 2. when a suggestion is clicked, the right section will render the clicked suggestion's page.

Good idea @Phyozawhein! :)

**Comment by Kushagra102** [comment]
@CBID2 I would Like to work on this issue. I have experience building such a functionality.

**Comment by CBID2** [comment]
> @CBID2 I would Like to work on this issue. I have experience building such a functionality. 

It's already assigned to @Phyozawhein

**Comment by k-deepak04** [comment]
@Phyozawhein please update your progress.....and try to complete the issue as soon as possible :)

**Comment by Phyozawhein** [comment]
My apologies. I came across some unprecedented personal issues. I would like to be taken off of the issue and have others work on it.

**Comment by k-deepak04** [comment]
no worries
i'll take this issue on

@CBID2 please assign it to me....also one request this issue will use logic for suggestion and coding so i should level 1 should be upgraded too...

**Comment by rupali-codes** [comment]
@k-deepak04 thanks, lets go!

**Comment by k-deepak04** [comment]
yes this week i'll wind it up for sure. or if won't able to do so, i'll drop a msg :)

**Comment by k-deepak04** [comment]
> @k-deepak04 are you working on this?

as said i have completed the task :)
now review and merge that.

Also i can work on other issues tommorow, let me know if anything important is to be done.

**Comment by k-deepak04** [comment]
@CBID2 here you have added the level 3 and in pr level 2
so please look and assign the pr with level 3
Thanks :)
